app/models/profile.rb

```diff
@@ -0,0 +1,6 @@
+module Profile
+  extend ActiveSupport::Concern
+
+  included do
+  end
```

Comment:
Hey @joemasilotti I've got reliable internet again so am finally starting to sort through & digest this PR. If I'm reading this correctly, a user would be associated with various accounts (each of which has their own profile) but there would not actually be a profile (avatar/name/bio) for that user specifically? 

Doesn't that mean that the only way a user gets a contact name/avatar is through a developer or business account? (I realize that's how it's currently set up, I guess I'm just asking - is that the plan for the future as well?) Wouldn't that mean that all users associated with XYZ Business would appear to be that particular business as they acted in the app? (And that I'd have a different avatar and potentially a different name based on what context I was operating in?)

This might be the intended way for it to be set up. But it feels off to me that if I'm using RailsDevs I don't really 'exist' as a person other than through the accounts I'm associated with (and for some of those accounts I might appear to be a completely _different_ person than who I am - ie the main contact person for the business.)